public/app/plugins/datasource/influxdb/query_ctrl.ts

```diff
@@ -74,6 +78,15 @@ export class InfluxQueryCtrl extends QueryCtrl {
       fake: true,
       value: '-- remove tag filter --',
     });
+
+    // to be able to use lodash.throttle:
+    // 1. we need to call it in the constructor, so it is only called once,
+    //    and not on every method-call
+    // 2. we need to work in a way where there is no return-value from the function.
+    //    we achieve this by calling resolve/reject as callback
+    this.getMeasurementsCallback = throttle((filter: string, result: any, error: any) => {
```

Comment:
to be able to use `lodash.throttle`, we cannot have a data-returning function (because lodash will never return the result of the promise-that-was-run-at-the-very-end), so we have to wrap the promise into a callback-using-function.